packages/errors/src/errors/ResponseError.ts

```diff
@@ -34,7 +35,7 @@ export class ResponseError extends Error {
    * Note that the body of this response is always consumed. Its parsed form is
    * in the `body` field.
    */
-  readonly response: Response;
+  readonly response: ConsumedResponse;
```

Comment:
Hm see the comment above; the core field is actually `body` right, with `text` acting as a helper. And us reading `text` is kind of an implementation detail 🤔 Tricky. I presume you chose to make it like this since `body` is the one with the differing implementation between node and browsers.

Reply:
Yeah `text` felt like a good low-level enough method that was also common across envs. Not feeling we're likely to be going to need to handle binary data here in the future.

Reply:
Unless we did like `ConsumedResponse<TBody = unknown>` or whatnot. Yeah sure, keep it like this for now